### DISCUSSION TITLE
Feature Request: Publish UTM SE on the App Store

### DISCUSSION THREAD

**Discussion by drehgelenkbuchen** [utmapp/UTM#4577] [UTM Help (iOS)] [answered]
Hi there!

Is there actually anything stopping UTM SE to be released on the iPhone App Store? It does not do JIT, and there are Virtual Machines allowed on the iOS App Store already (iSH), so I think this should be ok?

Forgive my ignorance if this question sounds stupid.

**Comment by mygizli04** [comment]
There was a TestFlight before but Apple took it down, so I don't believe they'd accept it.

**Comment by osy** [comment, marked as answer]
Apple rejected it.

**Reply by TommyTran732** [reply]
Any chance you could try to publish it again?

There's this app called Merge VM, which is/was a fork of/repackaged UTM and it somehow managed to made its way to the App Store just fine.

**Reply by upintheairsheep** [reply]
why?

**Reply by TylerAustin7** [reply]
I have noticed this app is also on the appstore as well app cell Merge VM

any chance you can create a separate version for the iOS app store!

**Reply by osy** [reply]
Y'all really think you're telling me something I don't know well enough?

**Reply by tifasoftware** [reply]
There's like an appeal feature for App Store publishishg, right?

**Reply by osy** [reply]
We've gone through the appeals almost a dozen times. We've contacted multiple people at Apple. We've reached out directly and indirectly to developer relations. Anything you can think of, I assure you has been tried multiple times.

Honestly the only thing to do is for someone else unrelated to me creating a developer account and submitting UTM SE under their account with a new name for free (the same thing those guys did but for a profit). But don't coordinate with me because it's against App Store policy to have multiple accounts.

**Reply by RetroSoftwareRepository** [reply]
> Any chance you could try to publish it again?
> 
> 
> 
> There's this app called Merge VM, which is/was a fork of/repackaged UTM and it somehow managed to made its way to the App Store just fine.

Merge VM isn't available in the US AppStore for some reason.

**Comment by upintheairsheep** [comment]
> Apple rejected it.



> Any chance you could try to publish it again?
> 
> There's this app called Merge VM, which is/was a fork of/repackaged UTM and it somehow managed to made its way to the App Store just fine. 

Is it a UTM SE or UTM fork? But anyways I hope Apple takes that app down, thieves.

**Reply by ddnomad** [reply]
I wonder whether the recent change of stance on emulation would also make it possible for UTM to be published for iOS / iPadOS. 

Being able to run full fat windows for visual novels etc. on iPad without sideloading shenanigans sounds quite alluring.

**Comment by thegushi** [comment]
So, the thing to look at here is because QEMU is open-source software under the GPL.  Have a look at [this 2010 article](https://www.fsf.org/blogs/licensing/more-about-the-app-store-gpl-enforcement) and it should give you a good idea about why apple is unlikely to approve any code that's licensed under it.  Apple does not want the FSF suing them.

I believe this problem also existed around 2010-2011 for VLC (the media player) which was resolved by by-licensing it, as it presently *is* in an app store.

You can keep trying, but you're just wasting developer time here by asking.

**Reply by Proryanator** [reply]
Was going to ask this myself, figured Apple would reject it.

**Reply by Grimsgit** [reply]
Well, then let’s license it properly so we can release it? There’s enough demand for this project to justify it. (Please don’t tear me to shreds.)
